src/main/java/org/dasein/cloud/network/VPNSupport.java

```diff
@@ -66,7 +66,7 @@
     @Deprecated
     public @Nonnull VPN createVPN(@Nullable String inProviderDataCenterId, @Nonnull String name, @Nonnull String description, @Nonnull VPNProtocol protocol) throws CloudException, InternalException;
     
-    public @Nonnull VPN createVPN(@Nonnull VpnLaunchOptions vpnLaunchOptions) throws CloudException, InternalException;
+    public @Nonnull VPN createVPN(@Nonnull VpnCreateOptions vpnLaunchOptions) throws CloudException, InternalException;
```

Comment:
Could you clean up the 'public' modifier from this interface methods while you're at it, please?